.travis.yml

```diff
@@ -27,13 +27,13 @@ before_install:
     - pip install numpydoc
 
 install:
-    - pip install -e .[display]
+    - pip install -e .[display,testing]
 
 before_script:
     - pep8 mir_eval evaluators tests
 
 script:
-    - nosetests -v --with-coverage --cover-package=mir_eval
+    - nosetests -v --with-coverage --cover-package=mir_eval -w tests
```

Comment:
Yep.  This way you can also execute from within the `tests` directory, and the tests don't have to know that they live within the `tests/` directory of the repo.